enterprise/backend/src/metabase_enterprise/content_translation/api/dictionary.clj

```diff
@@ -93,6 +110,7 @@
   "Import translations from CSV and insert or update rows in the content_translation table."
   [{:keys [file]}]
```

Comment:
right now we're getting the file like `(get-in multipart-params ["file" :tempfile])`, pretty sure we can do `(get-in multipart-params ["file" :size])` here.

For extra credit you could change `metabase.api.macros/middleware-forms` to allow configuring the arguments to `ring.middleware.multipart-params/wrap-multipart-params`. That allows us to set a max file size, which I think will short-circuit if the size is surpassed. (feels like an easy DOS if we allow people to upload a 10G file... actually wonder if we should do this by default every time we add `wrap-multipart-params` middleware... maybe there's not much point in doing it here if there are 10 other places we're accepting arbitrarily large multipart uploads)

Reply:
See https://ring-clojure.github.io/ring/ring.middleware.multipart-params.html and https://ring-clojure.github.io/ring/ring.middleware.multipart-params.temp-file.html